### PR TITLE
Clarification regarding Alexa's proactive mode

### PR DESCRIPTION
**Pull request by Molodax**
Clarification regarding necessity to restart HA as well as modified lambda function are added following the issue https://github.com/home-assistant/home-assistant/issues/24867#issuecomment-509063673

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9863"></a>

### CODE REVIEW

File: source/_components/alexa.smart_home.markdown [hunk 1]
```diff
@@ -93,7 +93,7 @@ Next you need create a Lambda function.
 - Under `Configuration` tab, expand `Designer`, then click `Alexa Smart Home` in the left part of the panel to add a Alexa Smart Home trigger to your Lambda function.
 - Scroll down little bit, you need input the `Skill ID` from the skill you created in previous step. (tips: you may need switch back to Alexa Developer Console to copy the `Skill ID`.
 - Click your Lambda Function icon in the middle of the diagram, scroll down you will see a `Function code` window.
-- Clear the example code, copy the Python script from: <https://gist.github.com/awarecan/630510a9742f5f8901b5ab284c25e912>
+- Clear the example code, copy the Python script from: <https://gist.github.com/awarecan/630510a9742f5f8901b5ab284c25e912> (doesn't support Alexa's proactive mode) or <https://gist.github.com/matt2005/744b5ef548cc13d88d0569eea65f5e5b> (modified code to support Alexa's proactive mode, see details below)
```
Comment (balloob) [inline source/_components/alexa.smart_home.markdown]: We should just link to the fixed one.

File: source/_components/alexa.smart_home.markdown [hunk 2]
```diff
@@ -212,12 +212,15 @@ alexa:
...
+The `endpoint`, `client_id` and `client_secret` are optional, and are only required if you want to enable Alexa's proactive mode (i.e. "Send Alexa Events" enabled). Please note the following if you want to enable proactive mode:
 
 - There are different endpoint URLs, depending on the region of your skill. Please check the available endpoints at <https://developer.amazon.com/docs/smarthome/send-events-to-the-alexa-event-gateway.html#endpoints>
 - The `client_id` and `client_secret` are not the ones used by the skill that have been set up using "Login with Amazon" (in the [Alexa Developer Console][amazon-dev-console]: Build > Account Linking), but rather from the "Alexa Skill Messaging" (in the Alexa Developer Console: Build > Permissions > Alexa Skill Messaging). To get them, you need to enable the "Send Alexa Events" permission.
 - If the "Send Alexa Events" permission was not enabled previously, you need to unlink and relink the skill using the Alexa App, or else Home Assistant will show the following error: "Token invalid and no refresh token available."
 
+<p class='note warning'>
```
Comment (balloob) [inline source/_components/alexa.smart_home.markdown]: Remove this warning. This should be covered by the instructions instead.

Comment (Molodax) [reply source/_components/alexa.smart_home.markdown]: I removed the warning and added a sentence to the instruction.

**Review by frenck — APPROVED** [review]
Thanks for updating the PR @Molodax! 👍

**Pull request merged**